src/nrnoc/solve.cpp

```diff
@@ -393,7 +393,7 @@ void nrn_solve(NrnThread* _nt) {
 #endif
 }
 
-#if VECTORIZE && _CRAY
+#if _CRAY
```

Comment:
this is for next PR but I believe `_CRAY` is not used any longer as well. But lets keep that for separate PR.